rasa/test.py

```diff
@@ -131,15 +131,20 @@ def test_core(
         )
         return
 
-    _agent = Agent.load(unpacked_model)
+    core_path, nlu_path = rasa.model.get_model_subdirectories(unpacked_model)
 
-    if _agent.policy_ensemble is None:
+    if not core_path:
         cli_utils.print_error(
             "Unable to test: could not find a Core model. Use 'rasa train' to train a "
             "Rasa model and provide it via the '--model' argument."
         )
 
-    if isinstance(_agent.interpreter, RegexInterpreter):
+    use_e2e = additional_arguments.get("e2e", False)
+
+    _interpreter = RegexInterpreter()
+    if nlu_path:
+        _interpreter = NaturalLanguageInterpreter.create(nlu_path)
+    elif use_e2e:
```

Comment:
I don't think this should be changed, this looks like a merge error, the only difference is the if condition